src/overlay/gather-markers.ts

```diff
@@ -500,7 +501,20 @@ class OutputMarker {
     this._gatherLabel.node.textContent = 'Gather';
     (this._gatherButton.layout as PanelLayout).addWidget(this._gatherLabel);
 
+    $(this._element).css({ overflow: 'visible' });
+    
+    let element = this._element;
+    for (var i = 0; i < 3; i += 1) {
```

Comment:
@jasonsjiang Can you give a name to this magic number `3`? What does it represent? Should this instead iterate over parents until it reaches parents with a specific class?

Reply:
<img width="414" alt="Screen Shot 2019-07-18 at 1 30 57 PM" src="https://user-images.githubusercontent.com/34109795/61482515-6d774b80-a960-11e9-999d-4a2f054db04b.png">
The `p-Widget` class needs to be removed for three parents of the gather button so that it will show up properly for the data frame output; otherwise, `p-Widget` will hide the overflow.